src/Common/tests/System/Collections/TestingTypes.cs

```diff
@@ -147,6 +147,9 @@ public int GetHashCode(int x)
 
     #region TestClasses
 
+#if netstandard17
```

Comment:
No strong opinion.  If you'd prefer I switch to such a dummy type, I can.  Are you interested in that just for the tests, or for the product source as well?